src/components/projects/CreateProjectForm.vue

```diff
@@ -917,8 +1063,21 @@ export default {
     this.targets[0].ra = this.mount_ra
     this.targets[0].dec = this.mount_dec
     this.targets[0].name = this.mount_object
-    this.targets.RAhours = true
-    this.targets.hrsminssecs = false
+
+    // initialize smart stack and long stack to camera tab values
+    this.project_constraints.longstack = this.longstackIsActive
+    this.project_constraints.smartstack = this.smartstackIsActive
+  },
+  created () {
+    // initialize expiry date to one lunar month from now, and start date to today
+    const today = new Date()
+
+    this.project_constraints.expiry_date.setDate(today.getDate() + 28)
+    this.project_constraints.start_date.setDate(today.getDate())
+
+    // converting from user's timezone to UTC
+    this.project_constraints.expiry_date.setMinutes(this.project_constraints.expiry_date.getMinutes() + today.getTimezoneOffset())
+    this.project_constraints.start_date.setMinutes(this.project_constraints.start_date.getMinutes() + today.getTimezoneOffset())
```

Comment:
Best to not use expiry_date and start_date as temporary variables (line 1075-76) used in creating the actual expiry_date and start_date (lines 1079-80). Instead, just define new variables for line 1075-6 and then you can use them for calculating the final value used to set `this.project_constraints.expiry_date`, etc.